packages/react-components/style/output-components-style.css

```diff
@@ -248,3 +253,21 @@ canvas {
 .hoverClass:hover {
     border: 1px solid black;
 }
+
+.options-menu-drop-down {
+    position: absolute;
```

Comment:
It seems to me that as a menu popup, the dialog should appear in a position relative to the mouse cursor or the menu icon.

Especially that where it is placed now, and with no user indication of how to close the dialog (click outside?), the user might mistakenly think that the [x] close button is to close the dialog, but it closes the table output.